docs/important/relevant_functionalities_en.md

```diff
@@ -1254,4 +1295,10 @@ The set or default value from `dogu.json` can be retrieved as usual. `doguctl` t
 
 ```
 rootLogLevel=$(doguctl config logging/root)
-```
\ No newline at end of file
+```
+
+#### Extensibility
+
+If there is a need for additional, Dogu-specific log levels, it is conceivable that the associated keys can also be stored under `/config/<doguname>/logging/`.
```

Comment:
This paragraph reads awkward. I'll suggest a sentence which may hopefully be better to grasp.
```suggestion
If there is a need for additional Dogu-specific log levels, the dogu root log level can be supplemented with additional keys (also under `/config/<doguname>/logging/`).
```